.release-please-manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+	"packages/logger": "4.0.2",
```

Comment:
I'd say we should adjust the versions accordingly, before merging this PR. Otherwise release please will create release PRs for v4, which can be very confusing.